build.sbt

```diff
@@ -261,7 +249,6 @@ lazy val docs = project
     inConfig(Compile)(defaultParadoxSettings),
     ParadoxPlugin.paradoxSettings(Local),
     inConfig(Local)(defaultParadoxSettings),
-    paradoxGroups := Map("Language" -> Seq("Scala", "Java")),
     paradoxProperties in Local ++= Map(
       // point API doc links to locally generated API docs
       "scaladoc.akka.stream.alpakka.base_url" -> rebase(
```

Comment:
Lots of unrelated changes here that should not be in the PR

Reply:
sorry ... a myopic merge :(